webapp/components/post_view/components/post_message_view.jsx

```diff
@@ -111,14 +113,17 @@ export default class PostMessageView extends React.Component {
             return <div>{renderedSystemMessage}</div>;
         }
 
+        const htmlFormattedText = TextFormatting.formatText(this.props.post.message, options);
+        const postMessageComponent = Utils.postMessageHtmlToComponent(htmlFormattedText, AtMentionProfile, this.props.usernameMap);
```

Comment:
Can you make `Utils.postMessageHtmlToComponent` a method of the `PostMessageView` instead? It's only used here for now, so I don't think it belong in utils